### DISCUSSION TITLE
Default SSH username and password not applying to groups connections or connections in general

### DISCUSSION THREAD

**Discussion by pambos1337** [Eugeny/tabby#9213] [General]
On default profile group settings for an SSH group I set up a default SSH profile with username and password .But when i connect to one of the SSH connections of that group it is still asking me to type the username and password again. Why is it not using the user and password from the group i set?
I also tried to set up a default profile for all SSH connections but again it's asking me every time.

**Comment by ChishFoxcat** [comment]
I'm in the same situation, it's more obvious on Tabby Web.

**Reply by pejahlo** [reply]
Same form me, I would appreciate if pwd change in default ssh profile is being reflected in all profiles using default profile.

**Reply by mmm286** [reply]
Same form me, I would appreciate if pwd change in default ssh profile is being reflected in all profiles using default profile.

**Comment by ianaflous** [comment]
I suggested this https://github.com/Eugeny/tabby/pull/10076. It's not using groups specifically, but it will prevent saving the same credentials in many profiles. It has been merged, it's available in the [Latest release](https://github.com/Eugeny/tabby/releases/latest).
